getting-started.md

```diff
@@ -27,8 +27,7 @@ Let's start with the basics.
      response.end('Hello World!\n');
   });
 
-  // Try retrieving a port from an environment variable, otherwise fallback to 8080.
-  var port = process.env.PORT || 8080;
+  var port = 3000;
```

Comment:
btw, probably it should be 8080 here right?
anyway, process.env will be missed if one want to `npm test` the s out of this